eZ/Bundle/PlatformInstallerBundle/src/Event/Subscriber/BuildSchemaSubscriber.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\PlatformInstallerBundle\Event\Subscriber;
+
+use EzSystems\DoctrineSchema\API\Event\SchemaBuilderEvent;
+use EzSystems\DoctrineSchema\API\Event\SchemaBuilderEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class BuildSchemaSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var string
+     */
+    private $schemaFilePath;
+
+    public function __construct(string $schemaFilePath)
+    {
+        $this->schemaFilePath = $schemaFilePath;
+    }
+
+    /**
+     * Returns an array of events this subscriber wants to listen to.
+     *
+     * @return string[]
```

Comment:
nitpick: this does not seems right in this case

Reply:
Whoops, good catch. Fixed it and added some strict return types, squashed into 6c08784.